vendor/bat-native-ledger/test/data/publisher_info_db_v30.sql

```diff
@@ -205,7 +205,6 @@ CREATE TABLE IF NOT EXISTS "event_log" (
 INSERT INTO "meta" VALUES ('mmap_status','-1'),
  ('version','30'),
  ('last_compatible_version','1');
-INSERT INTO "pending_contribution" VALUES (1,'pub_key', 1.0, 0, 'v01', 0);
```

Comment:
Curious: why was this line removed?

Reply:
I found a simpler way to check whether the `pending_contribution` table has the `processor` column:
```
EXPECT_TRUE(GetDB()->DoesColumnExist("pending_contribution", "processor"));
```